app/src/main/java/com/kimdoori/githubapp/binding_adapter/ViewBindingAdapter.kt

```diff
@@ -0,0 +1,13 @@
+package com.kimdoori.githubapp.binding_adapter
+
+import android.view.View
+import androidx.databinding.BindingAdapter
+
+@BindingAdapter("bind_visibility")
+fun setVisibility(view: View, isVisible: Boolean) {
+    if (isVisible) {
+        view.visibility = View.VISIBLE
+    } else {
+        view.visibility = View.GONE
+    }
```

Comment:
```suggestion
    view.isVisible = isVisible
```

ktx 를 사용하시면 한 줄로도 가능합니다 😄

Reply:
감사합니다!! https://github.com/kimdoori/GitHubApp/pull/20/commits/f4c19da82221cc256ced2d45e79f596e59e7d0f3 반영했습니다~!